symbols/peak.svg

```diff
@@ -4,5 +4,5 @@
    version="1.1"
    width="8"
    height="8">
-	<path d="M 4,1 0,8 8,8 z" />
+	<path d="M 4,1 0,8 8,8 z" style="fill: #d08f55; fill-opacity: 1;" />
```

Comment:
Or is that not possible with shield symbolizers? there's `shield-fill: #d40000;` in the code